airbyte-integrations/connectors/source-omnisend/manifest.yaml

```
@@ -297,6 +321,19 @@ schemas:
         type:
           - string
           - "null"
+      customProperties:
+        type:
+          - object
+          - "null"
+        additionalProperties:
+          anyOf:
+            - type: string
+            - type: number
+            - type: boolean
+            - type: array
+              items:
+                type: string
+            - type: "null"
```

```suggestion
        type:
          - object
          - "null"
        additionalProperties:
         true
```

This way it will receive any field inside customProperties key

---

committed those changes!